src/riot-rs-rt/src/lib.rs

```diff
@@ -53,7 +53,9 @@ static ISR_STACK: [u8; ISR_STACKSIZE] = [0u8; ISR_STACKSIZE];
 fn panic(_info: &core::panic::PanicInfo) -> ! {
     #[cfg(not(feature = "silent-panic"))]
     {
-        println!("panic: {}\n", _info);
+        #[allow(clippy::used_underscore_binding)]
```

Comment:
I think the better resolution here is to s/_info/info and allow the unused variable (eg. by `let _ =`'ing it)